pkgs/development/libraries/simpleitk/default.nix

```diff
@@ -4,12 +4,10 @@ stdenv.mkDerivation rec {
   pname = "simpleitk";
   version = "2.2.1";
 
-  outputs = [ "out" "dev" ];
-
   src = fetchFromGitHub {
     owner = "SimpleITK";
     repo = "SimpleITK";
-    rev = "v${version}";
+    rev = "refs/tags/v${version}";
```

Comment:
Why was this changed exactly?  `"v${version}"` ought to work fine, and is more customary across nixpkgs to my knowledge.

Reply:
However, I haven't looked for a reference in the Nix manual for this.

Reply:
It looks like it's originally from this https://discourse.nixos.org/t/why-is-the-r-ryantm-bot-using-refs-tags-in-the-version/20519  And it has an entry here: https://nixos.org/manual/nixpkgs/stable/#fetchgit

Thanks for alerting me to this.